polyfills/String/prototype/padEnd/polyfill.js

```diff
@@ -0,0 +1,17 @@
+(function () {
+	Object.defineProperty(String.prototype, 'padEnd', {
+		configurable: true,
+		enumerable: false,
```

Comment:
@JakeChampion This is so that it doesn't then somehow get polyfilled twice? I don't understand why the property should be marked as non-enumerable? What doesn't that tell us?